Source/IntegrationTests/LitTests.cs

```diff
@@ -161,6 +157,12 @@ public class LitTests {
       Config = new LitTestConfiguration(substitutions, commands, features, DafnyDriver.ReferencedEnvironmentVariables);
     }
 
+    public static ILitCommand DafnyCommand(IEnumerable<string> arguments, LitTestConfiguration config, bool invokeDirectly) {
```

Comment:
I've been trying to make "xlit" generic enough that we could one day apply it to Boogie as well. Would you mind adding a TODO to make this slightly more generic, by defining an interface with the `MainWithWriters` method that other test suites could define as well? That way we could have a single `MainMethodWithWritersCommand` that would point to such a method instead of implicitly "the main method of this assembly".

Reply:
I'm not a fan of leaving TODOs in code. I think a GH issue would be better. Also I'm sure what you're describing